adapters/vrtcal/vrtcaltest/exemplary/web-simple-banner.json

```diff
@@ -0,0 +1,102 @@
+{
+  "mockBidRequest": {
+    "id": "test-request-id",
+    "imp": [
+      {
+        "id": "test-imp-id",
+        "banner": {
+          "format": [
+            {
+              "w": 300,
+              "h": 250
+            }
+          ]
+        },
+        "ext": {
+          "bidder": {
+            "Just_an_unused_vrtcal_param": "unused_data_for_this"
+          }
+        }
+      }
+    ],
+    "site": {
+      "id": "fake-site-id",
+      "page": "http://www.vrtcal.com"
+    }
+  },
+  "httpCalls": [
+    {
+      "expectedRequest": {
+        "uri": "http://rtb.vrtcal.com/bidder_prebid.vap?ssp=1804",
+        "body": {
+          "id": "test-request-id",
+          "imp": [
+            {
+              "id": "test-imp-id",
+              "banner": {
+                "format": [
+                  {
+                    "w": 300,
+                    "h": 250
+                  }
+                ]
+              },
+
+	      "ext": {
+		  "bidder": {
+		    "Just_an_unused_vrtcal_param": "unused_data_for_this"
+		  }
+	     }
+            }
```

Comment:
Nitpick: The identation seems to include tabs, can we replace them with spaces? Also, can identation be 2 spaces throughout the document?
![image](https://user-images.githubusercontent.com/15863840/176757879-1a91d35e-e6a3-41f9-8ace-0b8a11c37637.png)